Doc/library/typing.rst

```diff
@@ -500,6 +504,8 @@ The module defines the following classes, functions and decorators:
 
     A generic version of :class:`collections.abc.Reversible`.
 
+   .. versionadded:: 3.5.2
```

Comment:
@DimitrisJim is right: please don't go through and do a mass update of indentation. If you're fixing it for the docs around where you added versionadded then that's fine, but otherwise just leave it be.